lib/migration.js

```diff
@@ -701,6 +701,12 @@ function mixinMigration(MySQL, mysql) {
     var p = this.getModelDefinition(model).properties[prop];
     var line = this.columnDataType(model, prop) + ' ' +
       (this.isNullable(p) ? 'NULL' : 'NOT NULL');
+    if (typeof p.mysql !== 'undefined' && p.mysql.default && typeof p.mysql.default === 'string') {
+      line = line + ' DEFAULT ' +
+            (p.mysql.default.toUpperCase() === 'CURRENT_TIMESTAMP' || p.mysql.default.toLowerCase() === 'now' ?
+              'CURRENT_TIMESTAMP' : '"' + p.mysql.default + '"');
```

Comment:
I will add a `typeof` check for the value being a `number` and not stringify the value in this case.